### ISSUE TITLE
Incompatible with angular 10

### ISSUE THREAD

**Issue by JohnnyQQQQ** [tinesoft/ngx-cookieconsent#89]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### CookieConsent and Library Versions?
```
- cookieconsent version: 3.1.0
- ngx-cookieconsent version: master
```

### Angular, Node and al Versions?

Angular 10

### Repro steps

- ng add ngx-cookieconsent
- add a component with ngx-cookieconsent
- ng build --prod

### The log given by the failure

```log
error TS2307: Cannot find module 'ngx-cookieconsent' or its corresponding type declarations.
 11 import { NgcCookieConsentService } from 'ngx-cookieconsent';
```

### Desired functionality

Should compile without error

### Mention any other details that might be useful

I already fixed another bug related to angular 10 here https://github.com/tinesoft/ngx-cookieconsent/pull/86 . As the Project is not using angular-cli I didn't open a PR to update it as I don't have time to investigate what should be changed etc. 

I would recommend investigating if a switch to angular-cli can be made for the long run.

**Comment by juanAFernandez** [comment]
Checked with 10.0.9 and works fine.

**Comment by tinesoft** [comment]
Should be fixed by 0d36a5650f7fc479faf0de7422cb8eb69e34ef07
